packages/ng/department/service/department-v4.service.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ILuApiResponse, LuApiV3Service } from '@lucca-front/ng/api';
+import { ILuTree } from '@lucca-front/ng/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ILuDepartment } from '../department.model';
+import { ILuDepartmentService } from './department-service.model';
+import { IApiDepartment } from './department-v3.service';
+
+@Injectable()
+export class LuDepartmentService extends LuApiV3Service<ILuDepartment> implements ILuDepartmentService<ILuDepartment> {
```

Comment:
We could name this service `LuDepartmentV4Service`: users knows it is V4 and one day, a V5 will be created 🙈